docs/GUIDE.md

```diff
@@ -0,0 +1,175 @@
+# Welcome to the OVAL-visualization-as-graph guide
+
+This tool generates a tree graph from ARF xml report from OpenSCAP scan.
+
+## Prerequisites
+
+### Minimal requirements
+
+- **python3**
+- [lxml](https://pypi.org/project/lxml/)
+
+### Recommended requirements
+
+- includes minimal requirements
+- [inquirer](https://pypi.org/project/inquirer/)
+
+## Installation
+
+### Fedora 30 and later
+
+```bash
+sudo dnf install oval-graph
+# Install inquirer, if you want have nice cli features. (optional)
+sudo pip3 install inquirer
+```
+
+### Installation via [pip](https://pypi.org/project/oval-graph/)
+
+```bash
+# If you want to install oval-graph to $HOME/.local/bin, you have to run the below command:
+pip3 install --user oval-graph
+# Install inquirer, if you want have nice cli features. (optional)
+pip3 install --user inquirer
+
+# If you want to install oval-graph globally instead, you have to run the below commands as admin, e.g. on Linux:
+sudo pip3 install oval-graph
+# Install inquirer, if you want have nice cli features. (optional)
+sudo pip3 install inquirer
+
+```
+
+### Installation from source
+
+```bash
+git clone https://github.com/OpenSCAP/OVAL-visualization-as-graph.git
+cd OVAL-visualization-as-graph
+
+# Install without features (light version)
+sudo pip3 install .
+
+# OR
+
+# Install with nice cli features
+sudo pip3 install ".[niceCli]"
+```
+
+> If you don't want to install anything, look [here](#run-from-source).
+
+## Commands
+
+- [arf-to-graph](#arf-to-graph) Generates a tree graph from ARF xml report
+- [arf-to-json](#arf-to-json) Generates a JSON from ARF xml report
+- [json-to-graph](#json-to-graph) Generates a tree graph from JSON
+
+### Positional arguments
```

Comment:
It's probably OK to describe stable positional argument, but I would just tell users "run the utility with `-h`" to see what optional args do - why to repeat here what is already present in the source code in form of the help message? Plus you may want add/remove/modify opt args way more than pos args.